boards/arm/bl654_usb/Kconfig.defconfig

```diff
@@ -26,7 +26,10 @@ config FLASH_LOAD_OFFSET
 if USB_DEVICE_STACK
 
 config USB_UART_CONSOLE
-	default y
+	default y if !USB_DEVICE_BLUETOOTH
```

Comment:
Changes in this file should be in own commit, which fixes #39904.

Reply:
@lairdjm could you please do that as suggested by @jfischer-no?

Reply:
@carlescufi this is awaiting a USB fix from @jfischer-no

Reply:
For completeness, option USB_UART_CONSOLE is removed. Shield approach in #40645 was not accepted. I opened #40645 to continue the discussion.

Anyway, I do not want to accept any USB device support setting at board level anymore. The boards in Zephyr tree should be in minimal configuration, https://docs.zephyrproject.org/latest/guides/porting/board_porting.html#general-recommendations, so no configuration to enable USB support or USB device classes.